src/accordion/Accordion.js

```diff
@@ -41,18 +46,20 @@ export const Accordion = {
    * @param {object} section DOM element to toggle state class on
    * @returns {void}
   */
-  openAccordionSection(section) {
+  openAccordionSection(section, sectionContent) {
     section.setAttribute(`aria-expanded`, true)
     section.classList.add(this.openClass);
+    Velocity(sectionContent, `slideDown`, this.transitionDuration);
   },
 
   /**
    * Closes the given Accordion
    * @param {object} section DOM element to toggle state class on
    * @returns {void}
   */
-  closeAccordionSection(section) {
+  closeAccordionSection(section, sectionContent) {
     section.setAttribute(`aria-expanded`, false)
     section.classList.remove(this.openClass);
+    Velocity(sectionContent, `slideUp`, this.transitionDuration);
   },
 };
```

Comment:
This is likely another module that we'll want to turn into the View pattern since it will generated on the client. We'll make this call after the search prototype.